data/base-crafting.yaml

```diff
@@ -1192,6 +1192,80 @@ deeds:
     medium_number: 15
     large_number: 16
 recipe_parts:
+  wire brush:
+    Crossing:
+      part-room: 8776
+      part-number: 10
+    Leth Deriel:
+      part-room: 8776
+      part-number: 10
+    #Riverhaven:
```

Comment:
Not sure if these rooms are the same in all societies for `wire brush` and `flask of oil`

Reply:
@KatoakDR Do you know a good way to populate these?

Reply:
@robbintt I don't, at least not outside of manually visit the shops and noting their room and part numbers. I also don't do crafting/smithing/etc so I'm probably not a good resource. @Dartellum @wstampley any suggestions here? Thanks

Reply:
(sorry, reposting my comment to stay "in character", haha)

@robbintt I don't, at least not outside of manually visit the shops and noting their room and part numbers. I also don't do crafting/smithing/etc so I'm probably not a good resource. @Dartellum @wstampley any suggestions here? Thanks

Reply:
That's the only way I know to do it. Go there and see what kind of shop it is and what number it is.

Reply:
If these are unvalidated, they should not be kept in the file.

If we only know part of the information (e.g. room # but not part #) then I could see a case to include what we do know, but keep a commented-out version of the stuff we don't.